### DISCUSSION TITLE
virtualbox-extension-pack: Alternatives and Rationale for removal

### DISCUSSION THREAD

**Discussion by adyang** [Homebrew/discussions#4263] [Casks] [answered]
### Output of `brew config`

```shell
NIL
```

### Output of `brew doctor`

```shell
NIL
```

### Description of issue

In https://github.com/Homebrew/homebrew-cask/pull/136834, it is mentioned that the cask `virtualbox-extension-pack` has been removed. From the comment, there are somethings that are not very clear so would like to clarify:
1. What is the recommended alternative for automating installation of virtualbox extension pack (since the cask is removed)?
2. What was the rationale and discussion for removal (was it mainly due to the large number of issues created by it?)?

Thanks!

### Relevant casks

virtualbox-extension-pack

**Comment by vitorgalvao** [comment, marked as answer]
There is no recommended automated alternative. The conclusion we reached is that it was not viable for us to support it any longer, but anyone who wishes to do so via their own methods is welcome to.

The rationale for removal is three fourths of the linked comment; for more specifics you may wish to search the issues and PRs in the repo.

Some software is not viable as a cask or formulae because it does not respond well to being automatically installed, causing issues for itself or other apps. If some software is used by a tiny percentage of users but requires a disproportionate amount of maintainer time to manage, every other formulae and cask suffers as a result. As such, it is reasonable to remove the offending tool. Before that point we’ll have tried solutions to solve the issue, but not everything is viable.

Naturally, this doesn't preclude you from having the formulae/cask in your own tap and dealing with the breakage yourself.

**Reply by adyang** [reply]
Is it possible to add a link in the MR to this discussion? I think some people might want to know more (2 confused emoticons in the MR), but am unable to find all the links to the scattered discussions.

Thanks!
